### PR TITLE
Added replicaset tracking to metrics

### PR DESCRIPTION
**Pull request by nathan-weinberg**
See https://github.com/cloud-bulldozer/e2e-benchmarking/pull/615 for context

### CODE REVIEW

**Comment by rsevilla87** [comment]
ReplicaSets get created on a 1:1 ratio with deployments, are you sure we need this?

**Comment by nathan-weinberg** [comment]
@memodi See above, does this satisfy the requirement you've mentioned?

File: cmd/kube-burner/ocp-config/metrics-aggregated.yml [hunk 1]
```diff
@@ -124,6 +124,9 @@
 - query: sum(kube_node_status_condition{status="true"}) by (condition)
   metricName: nodeStatus
 
+- query: count(kube_replicaset_labels{})
+  metricName: replicaSets
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics-aggregated.yml:128]: Can we make this query instant?, I think we don't need it timebased

**Review by nathan-weinberg — COMMENTED** [review]
dsadas

File: cmd/kube-burner/ocp-config/metrics-aggregated.yml (hunk 1 above)
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics-aggregated.yml:128]: I think is better to make this query instant and update its name to be consistent with other object counts.

```suggestion
- query: count(kube_replicaset_labels{})
  metricName: replicaSetCount
  instant: true
```

File: cmd/kube-burner/ocp-config/metrics.yml [hunk 2]
```diff
@@ -115,6 +115,9 @@
 - query: sum(kube_node_status_condition{status="true"}) by (condition)
   metricName: nodeStatus
 
+- query: count(kube_replicaset_labels{})
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics.yml:118]: Same as above

**Comment by nathan-weinberg** [comment]
Thanks for the review @rsevilla87! Made the requested changes :+1:

**Review by vishnuchalla — APPROVED** [review]
lgtm!

**Pull request merged**